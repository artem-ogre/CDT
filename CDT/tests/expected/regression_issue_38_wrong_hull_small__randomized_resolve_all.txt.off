OFF
9 13 0
0.03684926901466977 -5.155200928836618 0
0.2190464061752402 -5.155200928836618 0
0.127947837594955 -4.997413579558766 0
0.15147567518991 -5.144230291488 0
0.10442 -5.06098 0
0.1228735248885 -5.12897066233 0
0.1322969035965 -5.141286788425 0
0.115882234089 -5.115648852375 0
0.13262891777369 -5.119598039304 0
3 0 1 6
3 3 8 6
3 2 0 4
3 3 6 1
3 7 8 4
3 8 3 4
3 1 4 3
3 0 7 4
3 2 4 1
3 7 5 8
3 7 0 5
3 5 6 8
3 5 0 6
