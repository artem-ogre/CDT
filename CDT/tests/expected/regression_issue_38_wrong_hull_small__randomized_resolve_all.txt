13
0 1 3   4294967295 7 1
0 3 6   0 9 4
0 4 2   5 6 4294967295
0 5 7   4 12 5
0 6 5   1 11 3
0 7 4   3 10 2
1 2 4   4294967295 2 7
1 4 3   6 8 0
3 4 8   7 10 9
3 8 6   8 11 1
4 7 8   5 12 8
5 6 8   4 9 12
5 8 7   11 10 3

0

0

0
